### ISSUE TITLE
Can't kill emacs, possibly because of autosave?

### ISSUE THREAD

**Issue by daraul** [hlissner/doom-emacs#5108]
### 

- [X] I have searched [the FAQ](https://doomemacs.org/docs/faq.org) and [the documentation](https://doomemacs.org/docs/getting_started.org) for solutions.
- [X] I have searched [the issue tracker](https://github.com/hlissner/doom-emacs/issues) for similar issues (the closed issues too).
- [X] I have searched [the Discourse](https://doomemacs.org/d/docs) for any errors and solutions.
- [X] I can reproduce my issue on [the latest commit](https://github.com/hlissner/doom-emacs/commit) of Doom Emacs.
- [X] I have read [How to Debug Issues](https://doomemacs.org/d/how2debug) and found no solution.

### What did you expect to happen?

I hit `SPC q K` to quit emacs completely. I expected emacs to close, and the server to be killed. 

### What actually happened?

I get the error `Wrong type argument: stringp, nilInvalid face attribute :background nil` in the minibuffer. See ST below:

```
Debugger entered--Lisp error: (wrong-type-argument stringp nil)
  file-exists-p(nil)
  savehist-save(t)
  savehist-autosave()
  kill-emacs()
  save-buffers-kill-emacs(nil)
  #<subr funcall-interactively>(save-buffers-kill-emacs nil)
  apply(#<subr funcall-interactively> (save-buffers-kill-emacs nil))
  funcall-interactively(save-buffers-kill-emacs nil)
  #<subr call-interactively>(save-buffers-kill-emacs nil nil)
  apply(#<subr call-interactively> (save-buffers-kill-emacs nil nil))
  explain-pause--wrap-call-interactively(#<subr call-interactively> save-buffers-kill-emacs nil nil)
  apply(explain-pause--wrap-call-interactively #<subr call-interactively> (save-buffers-kill-emacs nil nil))
  call-interactively(save-buffers-kill-emacs nil nil)
  command-execute(save-buffers-kill-emacs)
```

### Describe your attempts to resolve the issue

I've tried installing different versions of emacs, thinking it might be me (flatpack, snaps, from ubuntu repos, nix). The only version I haven't tried yet is building from source. 

I suspected that maybe doom was trying to save my current state into some file, and failing. I guessed that toggling =auto-save-visited-mode= after starting doom might help, it didn't seem to do anything.

### Steps to reproduce

1. (possibly) reinstall doom
2. Start emacs in daemon mode: `emacs --daemon`
3. Start the client: `emacsclient -nc`
4. Quit: `SPC q K`

### System Information

https://pastebin.com/k8ybGrTn

**Comment by hlissner** [comment]
This was due to a breaking commit upstream in `doom-themes`, which was fixed in https://github.com/hlissner/emacs-doom-themes/issues/604 a short while ago.

This won't be an issue for Doom users who have doom-themes pinned, but for those that have unpinned it, run `doom sync -u` to update the package.

**Comment by hlissner** [comment]
Ah, my bad, had my wires crossed. I meant to link https://github.com/hlissner/doom-emacs/issues/5104, which is likely the cause for your issue, and was fixed a short while ago in 005a71e. Update Doom and your issue should go away.

**Comment by daraul** [comment]
You are incredibly quick, that was it. Thanks for taking the time!
